.travis.yml

```diff
@@ -128,6 +128,7 @@ matrix:
 script:
   - set -e
   - gcov --version
+  - etc/travis_fastfail.sh
```

Comment:
nitpicky: could go before the `gcov --version`

Reply:
True, but I actually placed it there because I thought sometimes the extra info might come in handy. But actually, I think the `gcov --version` should just be moved into `etc/ci-prepare.sh`, where we already have some other debug output. I'll do that now.